README.md

```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
 > Description the project.
```

Comment:
- [ ] Great job adding the project name, I kindly suggest that you also add the description of the project and add information under the relevant headers.